File: app/src/main/java/com/mobile/history/HistoryPosterClickListener.java
package com.mobile.history;

import com.mobile.history.model.ReservationHistory;

/**
 * Created by o_vicarra on 3/27/18.
 */

public interface HistoryPosterClickListener {

    void onPosterClicked(int pos, ReservationHistory historyposter, boolean isRatingScreen);
}

File: app/src/main/java/com/mobile/history/HistoryPosterClickListener.kt
package com.mobile.history

import com.mobile.history.model.ReservationHistory

/**
 * Created by o_vicarra on 3/27/18.
 */

interface HistoryPosterClickListener {

    fun onPosterClicked(pos: Int, historyposter: ReservationHistory, isRatingScreen: Boolean)
}